Sources/JWTKeychain/Routes/UserRoutes.swift

```diff
@@ -10,42 +10,43 @@ public struct UserRoutes: RouteCollection {
     private let authMiddleware: Middleware
     private let jwtAuthMiddleware: JWTAuthMiddleware
     private let protectMiddleware: ProtectMiddleware
+    private let configuration: ConfigurationType
 
     public init(
         drop: Droplet,
         authMiddleware: Middleware = AuthMiddleware<User>(),
-        jwtAuthMiddleware: JWTAuthMiddleware = JWTAuthMiddleware(),
         protectMiddleware: ProtectMiddleware = ProtectMiddleware(
             error: Abort.custom(
                 status: .unauthorized,
                 message: Status.unauthorized.reasonPhrase
             )
         )
-    ) {
+    ) throws {
         self.drop = drop
         self.authMiddleware = authMiddleware
-        self.jwtAuthMiddleware = jwtAuthMiddleware
         self.protectMiddleware = protectMiddleware
+        self.configuration = try Configuration(drop: drop)
+        self.jwtAuthMiddleware = JWTAuthMiddleware(configuration: self.configuration)
```

Comment:
I can make the init method receive the `Configuration` and `JWTAuthMiddleware` instancies if needed.